website/docs/docs/dbt-cloud/release-notes/06-Aug-2022/ide-improvement-beta.md

```diff
@@ -6,6 +6,11 @@ sidebar_label: "Enhancement: New cloud IDE beta"
 tags: [Aug-16-2022]
 ---
 
+:::info Beta Feature 
+Read more about the [Cloud IDE beta](https://www.getdbt.com/blog/staging-highlights-the-latest-from-dbt-labs/) and [submit your expression of interest](https://docs.google.com/forms/d/e/1FAIpQLSdlU65gqTZPyGAUc16SkxqTc50NO9vdq_KGx1Mjm_4FB_97FA/viewform) to join the new Cloud IDE Beta group.
```

Comment:
I think we can standardize on lowercase "b" in beta unless it's the first word in a title. 
```suggestion
Read more about the [Cloud IDE beta](https://www.getdbt.com/blog/staging-highlights-the-latest-from-dbt-labs/) and [submit your expression of interest](https://docs.google.com/forms/d/e/1FAIpQLSdlU65gqTZPyGAUc16SkxqTc50NO9vdq_KGx1Mjm_4FB_97FA/viewform) to join the new Cloud IDE beta group.
```